ios/Buttercup/AppDelegate.m

```diff
@@ -27,6 +27,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
+
+  // Temporarily disable Dark Mode until we have it.
```

Comment:
What does this mean? Are we waiting on something in RN, our dependencies or is it a minimum-version issue?

Reply:
@perry-mitchell as we don't have dark mode colors yet in our codebase, we have to disable dark mode entirely because some elements (native alerts, search boxes etc) will automatically change to dark mode colors if it's enabled on iOS 13 and will look ugly. Hopefully we can enable dark mode soon. It just needs some stylesheet modifications

Reply:
Sounds good! Thanks